src/components/ImageFullScreen.tsx

```diff
@@ -15,11 +15,26 @@ const StyledImage = styled.img`
 const StyledWrapperImage = styled.div`
     display: flex;
     justify-content: center;
+    outline: none;
 `;
 const ImageFullScreen: React.FC<ImageFullScreenProps> = ({ src = '', alt = '' }) => {
+    const ref = useRef<HTMLDivElement>(null);
+
+    useClickOutside(ref, () => {
+        dispatchModalEvent(ModalEvent.ImageFullScreen)();
+    });
+
+    const [onESC] = useKeyboard([KeyCode.Escape], () => {
+        dispatchModalEvent(ModalEvent.ImageFullScreen)();
+    });
+
+    useEffect(() => {
+        setTimeout(() => ref.current && ref.current.focus(), 0);
```

Comment:
`ref.current?.focus()` ?

Reply:
tried, doesn't work, had to use setTimeout